### PR TITLE
[Debugger Plugin]: Add clickable links to Session.runs table

### PR DESCRIPTION
**Pull request by caisq**
Clicking a node or tensor in the table focuses on the corresponding node
in the graph visualizer, if it exists.

### CODE REVIEW

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html [hunk 1]
```diff
@@ -221,6 +221,13 @@ <h2>Continue...</h2>
         max-width: 540px;
         margin: 80px auto 0 auto;
       }
+      .sidebar {
+        position: relative;
+        height: 100%;
+        /* width: 40%; */
```
Comment (chihuahua) [inline tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html]: We can probably just delete these commented lines.

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html [hunk 2]
```diff
@@ -658,18 +666,45 @@ <h2>Continue...</h2>
...
-              this._activeRuntimeGraphDeviceName !== deviceName) {
-            this.set('_activeRuntimeGraphDeviceName', deviceName);
+          this._focusOnGraphNode(deviceName, nodeName);
+        });
+      },
+      _createFeedFetchTargetClickedHandler() {
+        return((graphElementName) => {
+          let nodeName = graphElementName;
+          // If it is a tensor name, get the node name.
+          if (nodeName.indexOf(':') !== -1) {
```
Comment (chihuahua) [inline tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html]: Optional: Maybe compute once and reuse value.

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html [hunk 3]
```diff
@@ -66,6 +69,11 @@
         background-color: rgb(172, 232, 188);
         font-weight: bold;
       }
+
+      .node-or-tensor-element {
+        text-decoration: underline;
+        cursor: pointer;
```
Comment (chihuahua) [inline tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html]: Optional: Add a `.node-or-tensor-element:hover` effect to emphasize how an item is clickable.

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html (hunk 1 above)
Comment (caisq) [reply tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html]: Done.

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html (hunk 2 above)
Comment (caisq) [reply tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html]: This computation is pretty cheap. So I'd rather save the space.

File: tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html (hunk 3 above)
Comment (caisq) [reply tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html]: Good suggestion. I did this for all clickable elements currently present in the Debugger Plugin.

**Pull request merged**